exercises/palindrome-products/palindrome_products_test.go

```diff
@@ -7,14 +7,15 @@ import (
 	"testing"
 )
 
-// API to impliment:
-// type Product struct {
-// 	Product int // palindromic, of course
-// 	// list of all possible two-factor factorizations of Product, within
-// 	// given limits, in order
-// 	Factorizations [][2]int
-// }
-// func Products(fmin, fmax int) (pmin, pmax Product, error)
+/* API to impliment:
```

Comment:
I'll actually go ahead and fix this. I think the rest of this is fine, so I'll get this merged today.